playbooks/vars/agent_configuration_vars.yaml

```diff
@@ -0,0 +1,6 @@
+# Enter the values for the agent configuration
+bigip_netloc: <bigip_ip_address_or_hostname>
+agent_ini_file: /etc/neutron/services/f5/f5-openstack-agent.ini
+undercloud: <true_or_false>     # Set to true by default. Set to false if using an undercloud deployment.
```

Comment:
See my previous comment in playbooks/roles/configure_lbaasv2_agent/vars/main.yml.  In fact, is the main.yml file necessary if this setting exists here?